### ISSUE TITLE
sign column background / signs background don't match

### ISSUE THREAD

**Issue by breuerfelix** [sainnhe/gruvbox-material#33]
![2020-04-18-164410_570x131_scrot](https://user-images.githubusercontent.com/36931468/79640810-ea578000-8193-11ea-89e8-b289d640d8a1.png)

there is a little bug with the signcolumn background. (it is not dedicated to gitgutter, also with coc etc)  
you can actually all my theme configuration on this screenshot.

it does not happen when setting transparent background tho.

my idea would be to make the whole sign column the lighter background (or make the signs the darker background but i prefer the first variant)

btw i tried to change the sign column background lighter but it seems like i just can't get it working
```
highlight SignColumn ctermbg=white guibg=white
```
i am having this as the last line in my vimrc so it should overwrite the colorscheme.  
are you able to help me out there ? white is just to test if i grab the correct highlight.

/edit when i enter the exact fix line as a command while nvim running it works. but even though its the last line in the vimrc it just doesnt work. any ideas ?
`autocmd VimEnter * highlight ...` also works... so strange

**Comment by sainnhe** [comment]
I have dealt with similar issues before, see [edge#6](https://github.com/sainnhe/edge/issues/6#issuecomment-570750204).

**Comment by breuerfelix** [comment]
https://github.com/airblade/vim-gitgutter/issues/696

just opened an issue, maybe they will face the problem. i am using coc-git now, thanks for the alternatives!

**Comment by airblade** [comment]
@breuerfelix This was fixed in airblade/vim-gitgutter@8db2fc5.

**Comment by clo4** [comment]
I came across this issue while trying stuff like `highlight clear SignColumn` which was resulting in the same thing as the screenshot.
For any future visitors that find this issue, if your goal is to _remove_ the background color from the sign column, gruvbox-material has a setting for this:
```viml
let g:gruvbox_material_sign_column_background = 'none'
" Make sure this goes before  :colorscheme gruvbox-material
```
(lesson learned; read the theme's documentation too 😆)
